js/wazirx.js

```diff
@@ -0,0 +1,709 @@
+                if (stopPrice === undefined) {
+                    throw new ArgumentsRequired (this.id + ' createOrder requires a stop-price argument for ' + type + ' orders');
+                }
+                request['stopPrice'] = stopPrice;
+                params = this.omit (params, [ 'stopPx', 'stopPrice' ]);
+            }
+        } else {
+            throw new ExchangeError (this.id + ' allows limit and stop-limit orders only');
+        }
+        const response = await this.spotV1PrivatePostOrder (this.extend (request, params));
+        // {
+        //     "id": 28,
+        //     "symbol": "wrxinr",
+        //     "price": "9293.0",
+        //     "origQty": "10.0",
+        //     "executedQty": "8.2",
+        //     "status": "wait",
+        //     "type": "limit",
+        //     "side": "sell",
+        //     "createdTime": 1499827319559,
+        //     "updatedTime": 1499827319559
+        // }
+        return this.parseOrder (response, market);
+    }
+
+    parseOrder (order, market = undefined) {
+        // {
+        //     "id":1949417813,
+        //     "symbol":"ltcusdt",
+        //     "type":"limit",
+        //     "side":"sell",
+        //     "status":"done",
+        //     "price":"146.2",
+        //     "origQty":"0.05",
+        //     "executedQty":"0.05",
+        //     "createdTime":1641252564000,
+        //     "updatedTime":1641252564000
+        // },
+        const created = this.safeInteger (order, 'createdTime');
+        const updated = this.safeInteger (order, 'updatedTime');
+        const marketId = this.safeString (order, 'symbol');
+        const symbol = this.safeSymbol (marketId, market);
+        const amount = this.safeString (order, 'quantity');
+        const filled = this.safeString (order, 'executedQty');
+        const status = this.parseOrderStatus (this.safeString (order, 'status'));
+        const id = this.safeString (order, 'id');
+        const price = this.safeString (order, 'price');
+        const type = this.safeStringLower (order, 'type');
+        const side = this.safeStringLower (order, 'side');
+        return this.safeOrder ({
+            'info': order,
+            'id': id,
+            'clientOrderId': undefined,
+            'timestamp': created,
+            'datetime': this.iso8601 (created),
+            'lastTradeTimestamp': updated,
+            'status': status,
+            'symbol': symbol,
+            'type': type,
+            'timeInForce': undefined,
+            'postOnly': undefined,
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'filled': filled,
+            'remaining': undefined,
+            'cost': undefined,
+            'fee': undefined,
+            'average': undefined,
+            'trades': [],
+        }, market);
+    }
+
+    parseOrderStatus (status) {
+        const statuses = {
+            'wait': 'open',
+            'done': 'closed',
```

Comment:
We're missing the `canceled` status here.

Reply:
@kroitor I've just mapped those that are different from the ccxt standard,  canceled it's the same so will be used as is

Reply:
@kroitor ups so probably I've missed it, going to add it thanks 🙂